pages/table/sticky-columns.page.tsx

```diff
@@ -190,6 +191,16 @@ export default () => {
             >
               Sorting disabled
             </Checkbox>
+
+            <Checkbox
+              checked={urlParams.enableKeyboardNavigation}
+              onChange={event => {
+                setUrlParams({ enableKeyboardNavigation: event.detail.checked });
```

Comment:
Shouldn't the `enableKeyboardNavigation` URL param be applied to the Table(s) below?

Reply:
It is, in this test page the urlSettings are passed to each table as `{...urlParams}`